doc/en/user/source/security/layer.rst

```diff
@@ -164,6 +164,9 @@ The mapping of roles to permissions is as follows:
      - r
      - r
 
+.. note:: Specific workspace rule will take precedence over the more generic rule.When *.*.r = * is set and more specific rule for workspace private is assigned as private.*.r = TRUSTED_ROLE,
+	Then specific rule for workspace would take precedence. Hence private workspace layers would be read by TRUSTED_ROLE users only and other users wont be able to read private workspace layers:
+
```

Comment:
```suggestion
.. note:: Specific workspace rule ``private.*.r=TRUSTED_ROLE`` will take precedence over the more generic rule ``*.*.r=*``. When a request is made to read a layer ``private:vulnerable_infrastructure`` the most specific rule available is used to control access. In this case the workspace rule ``private.*.r=TRUSTED_ROLE`` is the most specific and only users that have TRUSTED_ROLE will be granted ``r`` access and be able to read the ``private:vulnerable_infrastructure`` layer.  Other users that do not have the TRUSTED_ROLE will not be granted ``r`` access and will be unable to access the ``private:vulnerable_infrastructure`` layer.
```

Reply:
@jodygarnett : I have updated the table and notes accordingly

Reply:
Thanks!